src/app/pages/BlockDetailPage/index.tsx

```diff
@@ -46,11 +47,13 @@ export const BlockDetailPage: FC = () => {
             <dt>{t('common.block')}</dt>
             <dd>{block.round}</dd>
 
-            <dt>{t('common.age')}</dt>
+            <dt>{t('common.timestamp')}</dt>
             <dd>
+              {format(new Date(block.timestamp!), 'MMMM d, yyyy K:m aa OOO')} (
```

Comment:
bad minutes format: `April 23, 2022 4:7 PM GMT+2 (9 months ago)`

Reply:
Also, can you make this work with i18n instead? https://www.i18next.com/translation-function/formatting#datetime